accounts/models.py

```diff
@@ -27,12 +27,24 @@ def __str__(self):
         return str(self.user)
 
 
+# followee=俺がフォローしている人
+# follower=俺をフォローしている人
 # 新ユーザーの作成時に空のprofileも作成する
+
+
 @receiver(post_save, sender=User)
 def user_is_created(sender, instance, created, **kwargs):
     if created:
         Profile.objects.get_or_create(user=instance)
 
 
 class FriendShip(models.Model):
-    pass
+    follower = models.ForeignKey(
+        User, related_name="follower", on_delete=models.CASCADE, null=True
+    )
+    following = models.ForeignKey(
+        User, related_name="following", on_delete=models.CASCADE, null=True
+    )
+
+    def __str__(self):
+        return "{} : {}".format(self.follower.username, self.following.username)

```

Comment:
Python3.6以降はf文字列を使えるので，もし，特に理由がなければこちらを使いましょう！（他の箇所も同様）
```suggestion
        return f"{self.follower.username} : {self.following.username}"
```

Reply:
直しました